big_tests/tests/mam_helper.erl

```diff
@@ -950,7 +953,7 @@ parse_messages(Messages) ->
 
 bootstrap_archive(Config) ->
     random:seed(os:timestamp()),
-    Users = escalus_ct:get_config(escalus_users),
+    _Users = escalus_ct:get_config(escalus_users),
```

Comment:
Hmm... so we don't need this line at all, right?

Reply:
It's obviously a leftover from a debug print/log that somebody's forgotten to remove. I just figured it might be convenient to leave it in case of future debugging, but yes - let's remove it.